gc/base/SweepHeapSectioning.hpp

```diff
@@ -61,7 +63,7 @@ class MM_SweepHeapSectioning : public MM_BaseVirtual {
 	virtual void kill(MM_EnvironmentBase* env);
 
 	bool update(MM_EnvironmentBase* env);
-	virtual uintptr_t reassignChunks(MM_EnvironmentBase* env) = 0;
+	virtual uintptr_t reassignChunks(MM_EnvironmentBase* env);
```

Comment:
`virtual` might be removed later on at clean up stage as well

Reply:
yes, virtual is temporary for passing accepted build.